framework/src/mfem/bcs/MFEMVectorFunctorNormalIntegratedBC.C

```diff
@@ -0,0 +1,43 @@
+#ifdef MFEM_ENABLED
+
+#include "MFEMVectorFunctorNormalIntegratedBC.h"
+
+registerMooseObject("MooseApp", MFEMVectorFunctorNormalIntegratedBC);
+
+InputParameters
+MFEMVectorFunctorNormalIntegratedBC::validParams()
+{
+  InputParameters params = MFEMIntegratedBC::validParams();
+  params.addRequiredParam<MFEMVectorCoefficientName>(
+      "vector_coefficient",
+      "Vector coefficient whose normal component will be used in the integrated BC. A coefficient "
+      "can be any "
+      "of the following: a variable, an MFEM material property, a function, or a post-processor.");
+  return params;
+}
+
+// TODO: Currently assumes the vector function coefficient is 3D
```

Comment:
I should have deleted this in a previous PR. Tidying up now that I'm thinking of it.
```suggestion
```